src/services/plan-service.ts

```diff
@@ -296,4 +296,19 @@ export class PlanService implements PlanHierarchy {
         };
     }
   };
+
+  getPreSelectedPlanTypePerPathType = (planType: PlanType, pathType: PathType) => {
```

Comment:
I don´t think it's ok to add a mapper for old types to new in this service, perhaps in utils, like the rest of the mappers? 🤔

Reply:
@lucianaDilizio